src/glmfit.jl

```diff
@@ -31,6 +31,15 @@ end
 # returns the sum of the squared deviance residuals
 deviance(r::GlmResp) = sum(r.devresid)
 
+"""
+    updatemu!{T<:FPVector}(r::GlmResp{T}, linPr::T)
+
+Update the response type from the linear predictor.
+
+#### Arguments:
```

Comment:
I the recent [docstring style guide](http://docs.julialang.org/en/latest/manual/documentation/), we agreed on the principle that an `Arguments` section should only be used when there are so many of them that presenting them in the description isn't possible.

Here, I think it would be better to document them in the description, which is much shorter:
```
Update the response type `r` from the linear predictor `linPr`.
```


Also, isn't this the "GLM response" rather than the "GLM response type"?